public/app/features/alerting/unified/components/import-to-gma/NamespaceAndGroupFilter.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/css';
+import { useMemo } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { Field, Stack, VirtualizedSelect, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { useGetNameSpacesByDatasourceName } from '../rule-editor/useAlertRuleSuggestions';
+
+import { ImportFormValues } from './ImportFromDSRules';
+
+interface Props {
+  rulesSourceName: string;
+}
+
+export const NamespaceAndGroupFilter = ({ rulesSourceName }: Props) => {
```

Comment:
This is almost the same as `components/rule-editor/GroupAndNamespaceFields.tsx` - could we combine the two?

Reply:
Actually, looking at this more - we could probably move this to use `ComboBox` instead, as this one still lets you create a namespace/group by typing and pressing enter (which doesn't make sense in this context)